tests/trinity/json-fixtures-over-rpc/test_rpc_fixtures.py

```diff
@@ -184,21 +184,21 @@ def result_from_response(response_str):
     return (response.get('result', None), response.get('error', None))
 
 
-def call_rpc(rpc, method, params):
+async def call_rpc(rpc, method, params):
```

Comment:
All these became `async` because the `execute` of the RPC Server became async (due to the event-bused interaction and also because I think it's the right thing to do)